rgwadmin/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
empty file?

Reply:
Yes, just like other packages in the repo. I think it's needed to get auto-gen for the documentation.